QUEST.tsv

```diff
@@ -1407,96 +1408,96 @@ QUEST_20150317_001407	Sincerity of Moze
-QUEST_20150323_001435	It could be a broken plaster of the Goddess Statue to get rid of the monsters there. {nl}If I'm right then it must be Naktis who's behind it. {nl}I may be far away but I hear the stories. {nl}
-QUEST_20150323_001436	Naktis is looking for the clue to enter the Cathedral. {nl}If I'm right, not only the Forest of Prayer but all of the Pilgrim's Way must be like that. {nl}If your journey reaches the Cathedral, please get rid of Naktis. {nl}You have to stop the revelation from falling in the hands of demons.
-QUEST_20150323_001437	This place is over. {nl}All Tree Root Crystals are mutated. {nl}
-QUEST_20150323_001438	Some Tree Root Crystals are useless unless burned. {nl}Some even fume the core. {nl}
-QUEST_20150323_001439	We must do whatever it takes to turn it back but… now it's all tiresome…{nl}
-QUEST_20150323_001440	Feels like drowning in deep water. {nl}The Tree Root Crystal is slowly killing me. {nl}
-QUEST_20150323_001441	Must do whatever it takes to destroy it…{nl}Please save me.
-QUEST_20150323_001442	I have no desire. Am I dying now? {nl}Eating and even breathing feels meaningless. {nl}
-QUEST_20150323_001443	But it must go on.{nl}Unless that cursed Tree Root Crystal is destroyed…
-QUEST_20150323_001444	There are things that Witas cares about. {nl}Won't he get his mind back if you show him those? 
+QUEST_20150323_001432	Worship the Goddess Statue to receive blessings or move on to another area.
+QUEST_20150323_001433	What a nasty curse. {nl}Let's see which is stronger. The curse of Naktis or the purifier of the Goddess.
+QUEST_20150323_001434	Of course I have the purifier. {nl}But I won't give it unless you make a money offering.
+QUEST_20150323_001435	It could be a broken plaster of the Goddess Statue so get rid of the monsters there. {nl}If I'm right, then Naktis must be behind it. {nl}I may be far away but I hear the stories.
+QUEST_20150323_001436	Naktis is looking for a clue to enter the Cathedral. {nl}If I'm right, not only is the Forest of Prayer like that but all of Pilgrim Path. {nl}If you reach the Cathedral, please defeat Naktis. {nl}You have to stop the revelation from falling into demon hands.
+QUEST_20150323_001437	This place is dying. {nl}All the Tree Root Crystals have mutated.
+QUEST_20150323_001438	Some Tree Root Crystals are useless unless burned. {nl}Some even fume at the core.
+QUEST_20150323_001439	We must do whatever it takes to turn it back but… {nl}It's all so tiring now…
+QUEST_20150323_001440	I feel like I'm drowning. {nl}The Tree Root Crystal is slowly killing me.
+QUEST_20150323_001441	Do whatever it takes to destroy it… {nl}Please save me.
+QUEST_20150323_001442	I've lost all desire. Am I dying now? {nl}Eating and even breathing feels meaningless.
+QUEST_20150323_001443	But it must go on. {nl}Unless that cursed Tree Root Crystal is destroyed…
+QUEST_20150323_001444	There are things that Witas care about. {nl}Won't he get his mind back if you show him those? 
```

Comment:
This should be "cares".  Witas is a person ( at this point, a Pilgrim ), not a group of things.